benchmarks/jmh/src/jmh/kotlin/com/badoo/reaktive/benchmarks/jmh/scrabble/ScrabbleBase.kt

```diff
@@ -50,7 +50,7 @@ abstract class ScrabbleBase {
 
     private fun readResource(path: String) =
         BufferedReader(InputStreamReader(GZIPInputStream(this.javaClass.classLoader.getResourceAsStream(path)))).lines()
-            .map { it.toLowerCase() }.collect(Collectors.toSet())
+            .map { it.lowercase() }.collect(Collectors.toSet())
```

Comment:
`toLowerCase`/`toUpperCase` are now deprecated. Could not update the similar code in the `DarwinPlugin`, it does not see the new function and the compilation fails.

Reply:
Because for Gradle plugins I use built-in Kotlin version instead of the latest one. For `gradle-6.8.3` it is 1.4.